scripts/generator.ts

```diff
@@ -306,19 +307,26 @@ function generateReactComponent({ name, js }: SchemaHTMLElement, { packageName,
     `
 import type { EventName } from "${LIT_REACT_PATH}";
 import * as WebComponentModule from "${MODULE_PATH}";
+import { ${COMPONENT_NAME} as ${ELEMENT_CLASS_NAME} } from "${MODULE_PATH}";
 import * as React from "react";
 import { createComponent, WebComponentProps } from "${CREATE_COMPONENT_PATH}";
+
+export * from "${MODULE_PATH}";
+
+export {
+  /** @deprecated */WebComponentModule,
+  ${ELEMENT_CLASS_NAME},
+};
+
 export type ${EVENT_MAP};
 const events = ${EVENTS_DECLARATION} as ${EVENT_MAP_REF_IN_EVENTS};
-export type ${COMPONENT_NAME}Props = WebComponentProps<WebComponentModule.${COMPONENT_NAME}, ${EVENT_MAP}>;
+export type ${COMPONENT_NAME}Props = WebComponentProps<${ELEMENT_CLASS_NAME}, ${EVENT_MAP}>;
 export const ${COMPONENT_NAME} = createComponent({
-  elementClass: WebComponentModule.${COMPONENT_NAME},
+  elementClass: ${COMPONENT_NAME}Element,
```

Comment:
Wouldn't it be better to use `${COMPONENT_NAME}Element` everywhere instead of `ELEMENT_CLASS_NAME`?